sopel/cli/config.py

```diff
@@ -103,6 +108,8 @@ def handle_init(options):
 
     :param options: parsed arguments
     :type options: ``argparse.Namespace``
+    :return: Return 0 if everything went fine.
+             Return 1 if the file is invalid or if it already exists.
```

Comment:
```suggestion
    :return: 0 if everything went fine;
             1 if the file is invalid or if it already exists
```

Style-matching again. Kept your wrapping style though. 😁